Classes/common/CDTIncrementalStore.m

```diff
@@ -16,6 +16,14 @@
 #import "CDTFieldIndexer.h"
 #import "CDTISGraphviz.h"
 
+@interface CDTISBatchUpdateResult : NSPersistentStoreResult
+@property (nonatomic, strong) id result;
+@property (nonatomic, assign) NSBatchUpdateRequestResultType resultType;
```

Comment:
ACK.  I played around with this and found a variant that is concise and allows us to use the actual NSBatchUpdateResult class, so I'll commit that.